README.md

```diff
@@ -35,75 +35,104 @@ ctags --fields=afmikKlnsStz readtags.c  readtags.h
 **Opening Tags File**
 ```python
 import ctags
-from ctags import CTags, TagEntry
+from ctags import CTags
 import sys
 
 try:
     tagFile = CTags('tags')
-except:
+except OSError as err:
+    print err
```

Comment:
The "except OSError as err" is python3 syntax.
Do you speak about the print function ?